src/DarkModeColors.mdx

```diff
@@ -0,0 +1,176 @@
+import { Meta, ColorPalette, ColorItem } from '@storybook/blocks'
+
+<Meta title="Dark Mode Colors" />
+
+<ColorPalette>
+  <ColorItem
+    title="ds.pink"
+    subtitle="shades of pink"
+    colors={{
+      default: 'rgb(240,31,122)',
```

Comment:
I wonder if we could reference the tailwind css styles here. I'm not sure if you've already tried this, but it could be worth it. It might need some config changes in storybook though.